README.md

```diff
@@ -1,13 +1,17 @@
 # HOPS FLASK API
 
 # v.1.0
-### The root endpoint:
+
+## The root endpoint
```

Comment:
asdf

Reply:
I would not fix that because it is awesome!